src/ui.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Container,
+  render,
+  Text,
+  VerticalSpace,Code,Space
+} from "@create-figma-plugin/ui";
+import { h } from "preact";
+import { useEffect } from "preact/hooks";
+import {copyToClipboard} from 'figx'
+
+function UI({ content }: { content: string} ) {
+  useEffect(() => {
+    copyToClipboard(content)
+  }, [content])
+
+  return (
+    <Container space="medium">
+      <VerticalSpace space="medium" />
+        <Text>Copied to Clipboard.</Text>
+        <br />
+        <Code>{content}</Code>
```

Comment:
**Nitpick**: In their examples, [they always use the Code component inside Text](https://yuanqing.github.io/create-figma-plugin/storybook/?path=/story/inline-text-code--default), but I still have to test the difference.